bin/client/main.rs

```diff
@@ -106,10 +106,25 @@ async fn run() -> std::result::Result<(), ClientError> {
         }
         SubCommand::Search(search_opts) => {
             // TODO: Do we want to use the cache for searching?
-            let matches = bindle_client.query_invoices(search_opts.into()).await?;
-            tokio::io::stdout()
-                .write_all(&toml::to_vec(&matches)?)
+            let matches = bindle_client
+                .query_invoices(search_opts.clone().into())
                 .await?;
+
+            match search_opts.output {
+                Some(format) if format == "toml".to_owned() => {
```

Comment:
Nit: Pretty sure you can compare this by doing `&format == "toml"` so you don't have to get a string.

Reply:
Oh, yeah... that totally makes sense.